boot/bootutil/src/encrypted_priv.h

```diff
@@ -0,0 +1,82 @@
+
```

Comment:
Please add a license header. The short one is adequate, as in `boot/bootutil/src/encrypt_misc.c` for example.